gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/StorageAccountController.java

```diff
@@ -28,6 +28,7 @@
 
 @RestController
 @RequestMapping("/storage")
+@Deprecated
```

Comment:
Is there an accompanying change coming in Deck for this?

Reply:
Yup